spec/datetimeformat.html

```diff
@@ -203,7 +203,7 @@ <h1>Internal slots</h1>
           [[LocaleData]].[[&lt;_locale_&gt;]].[[hc]] must be &laquo; *null*, *"h11"*, *"h12"*, *"h23"*, *"h24"* &raquo;.
         </li>
         <li>
-          [[LocaleData]].[[&lt;_locale_&gt;]].[[hourCycle]] must be a String value equal to *"h11"*, *"h12"*, *"h23"*, or *"h24"*.
```

Comment:
This one seems to be imbalanced. I suppose there is an access to this `[[hourCycle]]` field that needs to be updated too?

Reply:
generally every commit landed to main needs to be correct, so it might need to all go in the same PR